cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -141,11 +133,16 @@ def test_create_without_description(small_test_vm):
         initialEstimate: 1/4h
         casecomponent: Infra
     """
-    snapshot = new_snapshot(small_test_vm, has_name=False, create_description=False)
-    with pytest.raises(AssertionError):
-        snapshot.create()
-    view = snapshot.parent_vm.create_view(InfraVmSnapshotAddView)
-    view.flash.assert_message('Description is required')
+    if small_test_vm.appliance.version >= '5.10':
+        # In 5.10 it's not possible to create a snapshot w/o description,"Create" button is disabled
+        view = navigate_to(small_test_vm, 'SnapshotsAdd')
+        assert view.create.disabled
```

Comment:
I think you should assert that the description field is empty before asserting that the create button is disabled.